README.md

```diff
@@ -1,4 +1,4 @@
-# Raft
+# Raft-rs
```

Comment:
The crate name is [raft](https://crates.io/crates/raft), so I think it's ok to not have the `-rs` suffix here.

Reply:
Yes, you are right.